python310Packages.ecs-logging: 2.0.2 -> 2.1.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python310Packages.ecs-logging is: Logging formatters for the Elastic Common Schema (ECS) in Python

meta.homepage for python310Packages.ecs-logging is: https://github.com/elastic/ecs-logging-python




###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 2.1.0 with grep in /nix/store/j0zk7h0l9cj4pzwyfgmh7kh25y3clwgi-python3.10-ecs-logging-2.1.0
- found 2.1.0 in filename of file in /nix/store/j0zk7h0l9cj4pzwyfgmh7kh25y3clwgi-python3.10-ecs-logging-2.1.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
5 total rebuild path(s)

5 package rebuild(s)

First fifty rebuilds by attrpath
elasticsearch-curator
python310Packages.ecs-logging
python310Packages.elastic-apm
python311Packages.ecs-logging
python311Packages.elastic-apm
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/j0zk7h0l9cj4pzwyfgmh7kh25y3clwgi-python3.10-ecs-logging-2.1.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python310Packages.ecs-logging https://github.com/r-ryantm/nixpkgs/archive/ba50fb0bbb47285dba05317e9f3d95d11288814c.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/j0zk7h0l9cj4pzwyfgmh7kh25y3clwgi-python3.10-ecs-logging-2.1.0
ls -la /nix/store/j0zk7h0l9cj4pzwyfgmh7kh25y3clwgi-python3.10-ecs-logging-2.1.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>elasticsearch-curator</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.ecs-logging</li>
    <li>python310Packages.elastic-apm</li>
    <li>python311Packages.ecs-logging</li>
    <li>python311Packages.elastic-apm</li>
  </ul>
</details>

---

###### Maintainer pings

cc @fabaff for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
python310Packages.ecs-logging: 2.0.2 -> 2.1.0

matthiasbeyer commented:
Result of `nixpkgs-review pr 249728` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>elasticsearch-curator</li>
    <li>elasticsearch-curator.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.ecs-logging</li>
    <li>python310Packages.ecs-logging.dist</li>
    <li>python310Packages.elastic-apm</li>
    <li>python310Packages.elastic-apm.dist</li>
    <li>python311Packages.ecs-logging</li>
    <li>python311Packages.ecs-logging.dist</li>
    <li>python311Packages.elastic-apm</li>
    <li>python311Packages.elastic-apm.dist</li>
  </ul>
</details>

anthonyroussel reviewed (approved):
LGTM :heavy_check_mark: 

Just 2 builds which are already failing on Hydra on the master branch.

Result of `nixpkgs-review pr 249728` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>2 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>elasticsearch-curator: log was empty</li>
<li>python311Packages.elastic-apm: log was empty</li>
  </ul>
</details>
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>elasticsearch-curator.dist: log was empty</li>
<li>python311Packages.elastic-apm.dist: log was empty</li>
  </ul>
</details>
<details>
<summary>
6 packages built:
</summary>
<ul>
<li>python310Packages.ecs-logging</li>
<li>python310Packages.ecs-logging.dist</li>
<li>python310Packages.elastic-apm</li>
<li>python310Packages.elastic-apm.dist</li>
<li>python311Packages.ecs-logging</li>
<li>python311Packages.ecs-logging.dist</li>
</ul>
</details>

fabaff committed:
python311Packages.es-client: 8.7.0 -> 8.9.0

Diff: https://github.com/untergeek/es_client/compare/refs/tags/v8.7.0...v8.9.0

Changelog: https://github.com/untergeek/es_client/releases/tag/v8.9.0

fabaff committed:
python311Packages.elasticsearch8: 8.7.0 -> 8.9.0

Changelog: https://github.com/elastic/elasticsearch-py/releases/tag/v8.9.0

fabaff committed:
elasticsearch-curator: 8.0.4 -> 8.0.8

Changelog: https://github.com/elastic/curator/releases/tag/v8.0.8

fabaff commented:
Result of `nixpkgs-review pr 249728` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>18 packages built:</summary>
  <ul>
    <li>elasticsearch-curator</li>
    <li>elasticsearch-curator.dist</li>
    <li>python310Packages.ecs-logging</li>
    <li>python310Packages.ecs-logging.dist</li>
    <li>python310Packages.elastic-apm</li>
    <li>python310Packages.elastic-apm.dist</li>
    <li>python310Packages.elasticsearch8</li>
    <li>python310Packages.elasticsearch8.dist</li>
    <li>python310Packages.es-client</li>
    <li>python310Packages.es-client.dist</li>
    <li>python311Packages.ecs-logging</li>
    <li>python311Packages.ecs-logging.dist</li>
    <li>python311Packages.elastic-apm</li>
    <li>python311Packages.elastic-apm.dist</li>
    <li>python311Packages.elasticsearch8</li>
    <li>python311Packages.elasticsearch8.dist</li>
    <li>python311Packages.es-client</li>
    <li>python311Packages.es-client.dist</li>
  </ul>
</details>

fabaff reviewed (approved):
Change looks good to me.
